packages/crayons-extended/custom-objects/src/components/form-builder/components/fb-field-dropdown-item.tsx

```diff
@@ -120,6 +137,18 @@ export class FbFieldDropdownItem {
     this.performLabelChange(event, false);
   };
 
+  private nameKeydownHandler = (event: KeyboardEvent) => {
+    const value = event?.target?.['value']?.trim() || '';
+    const keyEvent = event.detail?.['event'];
+    const isNextElement =
+      event?.target?.['parentNode']?.['parentNode']?.['parentNode']?.['host']?.[
```

Comment:
Have you checked if this condition is working for all combinations? This is not an ideal way to retrieve but its fine if its working

Reply:
It is working as expected in firefox, chrome. Will make sure the same case for safari as well.